apiserver/organization/organization_test.go

```diff
@@ -2,6 +2,8 @@ package organization
 
```

Comment:
This file has gotten quite large. Maybe we can split it up based on the verb, if that makes sense?
e.g. `organization_get_test.go`, `organization_update_test.go` etc.

Reply:
Yes that file got a bit large. I'll try to split it up.